src/core/Table/TablePaginator.tsx

```diff
@@ -311,6 +325,11 @@ export const TablePaginator = (props: TablePaginatorProps) => {
         </IconButton>
       </div>
       <div className='iui-right'>
+        {localization.rowsPerPageLabel && paginatorWidth >= 1024 && (
+          <Text isMuted as='span'>
+            {localization.rowsPerPageLabel}
```

Comment:
May need to do something to handle very long strings
![image](https://user-images.githubusercontent.com/88331924/142472088-3899f67c-b7cc-4cb2-8f97-b58f78a69751.png)
![image](https://user-images.githubusercontent.com/88331924/142472118-bdf8efd1-c5e1-4c29-9b92-0e906f1e1fed.png)

Reply:
this is easily fixed with `text-overflow: ellipsis` but that will need to be in CSS

Reply:
Added `.iui-paginator-page-size-label` from https://github.com/iTwin/iTwinUI/pull/381. Doesn't do anything currently as it needs CSS update (Not updating CSS in this PR because it will break buttons)